build/Microsoft.DotNet.Cli.DependencyVersions.props

```diff
@@ -1,11 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <CLI_MSBuild_Version>15.1.0-preview-000503-01</CLI_MSBuild_Version>
-    <CLI_Roslyn_Version>2.0.0-rc3-61212-03</CLI_Roslyn_Version>
-    <CLI_NETSDK_Version>1.0.0-alpha-20170105-5</CLI_NETSDK_Version>
-    <CLI_WEBSDK_Version>1.0.0-alpha-20170106-1-203</CLI_WEBSDK_Version>
-    <CLI_TestPlatform_Version>15.0.0-preview-20170106-08</CLI_TestPlatform_Version>
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CLI_MSBuild_Version>15.1.0-preview-000503-01</CLI_MSBuild_Version>
+    <CLI_Roslyn_Version>2.0.0-rc3-61212-03</CLI_Roslyn_Version>
+    <CLI_NuGet_Version>4.0.0-rc3-2193</CLI_NuGet_Version>

```

Comment:
@ericstj - done.

@weshaggard - yes.  Since this NuGet build was published to the `nuget-build` myget feed, it was automatically mirrored to the `cli-deps` feed.